tasks/main.yml

```diff
@@ -106,12 +106,75 @@
     group: "{{  odoo_role_odoo_group }}"
   notify: restart odoo
 
+- name: Check if Odoo database has been initialized ("0" false, "1" true)
+  become: yes
+  become_user: "{{ odoo_role_odoo_user }}"
+  shell: >
+    psql {{ odoo_role_odoo_db_name }} -tAc
+    "SELECT COUNT(*)
+    FROM information_schema.tables
+    WHERE table_name='ir_module_module';"
+  register: db_initialized
+  tags:
+    - mods
+
 - name: Init Odoo database
   become: yes
   become_user: "{{ odoo_role_odoo_user }}"
-  command: "{{ odoo_role_odoo_python_path }} {{ odoo_role_odoo_bin_path }} -c {{ odoo_role_odoo_config_path }}/odoo.conf -d {{ odoo_role_odoo_db_name }} --init {{ odoo_role_odoo_core_modules }} --stop-after-init --without-demo=all --logfile=/dev/stdout --log-level=warn"
+  command: >
+    {{ odoo_role_odoo_python_path }} {{ odoo_role_odoo_bin_path }}
+    -c {{ odoo_role_odoo_config_path }}/odoo.conf
+    -d {{ odoo_role_odoo_db_name }}
+    --init base
+    --stop-after-init
+    --without-demo=all
+    --logfile=/dev/stdout
+    --log-level=warn
+  when: db_initialized.stdout == "0"
   notify: restart odoo
+  tags:
+    - mods
 
 - import_tasks: community-modules.yml
 
+- debug:
+    msg: "{{ odoo_role_community_odoo_modules }}"
```

Comment:
I don't know why... but this variable doesn't get update when we modify it in `group_vars/all.yml` :rage4:

Reply:
Solved by https://gitlab.com/coopdevs/odoo-test-provisioning/merge_requests/2/diffs?commit_id=1f714f99ba57cbb17e8e97a1ee6048fdf0de2943